src/job/validate.ts

```diff
@@ -1,16 +1,18 @@
 import humanInterval from 'human-interval';
+import { parseExpression } from 'cron-parser';
 
-import { Job } from './Job';
 import { Logger } from '../logging/Logger';
 import { MomoErrorType } from '../logging/error/MomoErrorType';
 import { momoError } from '../logging/error/MomoError';
+import { isCronSchedule, isIntervalSchedule } from './MomoJob';
+import { Job } from './Job';
 
-export function validate({ name, interval, firstRunAfter, concurrency, maxRunning }: Job, logger?: Logger): boolean {
-  if (firstRunAfter < 0) {
+export function validate({ name, schedule, concurrency, maxRunning }: Job, logger?: Logger): boolean {
+  if (isIntervalSchedule(schedule) && schedule.firstRunAfter < 0) {
```

Comment:
maybe this check could be moved into the validateInterval function instead - so we have all checks for an interval based schedule in one method?